docs/DOCKER.md

```diff
@@ -29,7 +29,7 @@ Docker container, adding the appropriate command-line options as parameters.
 You first need to build the `lnd` docker image:
 
 ```shell
-⛰  docker build --tag=myrepository/lnd --build-arg checkout=v0.11.1-beta .
+⛰  docker build --tag=myrepository/lnd --build-arg checkout=v0.12.1-beta .
```

Comment:
Maybe it's worth waiting to update this once `v0.13.0-beta` is officially released?